ngrinder-frontend/src/js/components/script/List.vue

```diff
@@ -218,7 +218,14 @@
         }
 
         downloadScript(path) {
-            window.location.href = `${this.contextPath}/script/download/${path}`;
+            const link = document.createElement('a');
+            link.style.display = 'none';
+            link.href = `${this.contextPath}/script/api/download/${path}`;
+            link.setAttribute('download', path);
+
+            document.body.appendChild(link);
+            link.click();
+            document.body.removeChild(link);
```

Comment:
What is it for?

Reply:
Used it as a workaround for the MIME type warning. It's simpler to use `location.href` but, get a console warning like below.
```
Resource interpreted as Document but transferred with MIME type application/octet-stream ...
```
https://stackoverflow.com/questions/6587393/resource-interpreted-as-document-but-transferred-with-mime-type-application-zip/17452321#17452321

Reply:
okay